packages/playground/preload/router.js

```diff
@@ -6,7 +6,7 @@ const routes = [
   {
     path: '/about',
     name: 'About',
-    component: () => import('./src/components/About.vue')
+    component: () => import(/*breaks*/ './src/components/About.vue')
```

Comment:
Is the word `breaks` relevant? Otherwise we could use something like `/* a comment */`.

Also, instead of using this line/import, we could add a third route. Then we preserve the original test and add an additional test for this specific issue.

Reply:
Word `breaks` is irrelevant, `/* a comment */` is better!
I will add an additional test case.

Reply:
Done.